src/Types.hs

```diff
@@ -633,15 +633,11 @@ isWild (IdeaLocationTopic _ _) = False
 topicToIdeaLocation :: Topic -> IdeaLocation
 topicToIdeaLocation = IdeaLocationTopic <$> (^. topicIdeaSpace) <*> (^. _Id)
 
-userToIdeaLocation :: User -> IdeaLocation
-userToIdeaLocation u =
-    (\(Student clss) -> IdeaLocationSpace (ClassSpace clss)) . head $ u ^. userGroups
-
--- german group name
-groupLabel :: IsString s => Group -> s
-groupLabel (Student _)    = "Student"
-groupLabel (ClassGuest _) = "Gast (Klasse)"
-groupLabel SchoolGuest    = "Gast (Klasse)"
-groupLabel Moderator      = "Moderator"
-groupLabel Principal      = "Direktor"
-groupLabel Admin          = "Administrator"
+-- | german role name
+roleLabel :: IsString s => Role -> s
+roleLabel (Student _)    = "Student"
+roleLabel (ClassGuest _) = "Gast (Klasse)"
+roleLabel SchoolGuest    = "Gast (Klasse)"
```

Comment:
Just checking: should names these really be equal? No "Schule" in the second one?